backend/app/tests/api/routes/collections/test_create_collections.py

```
@@ -1,49 +1,137 @@
-from uuid import UUID
+from uuid import UUID, uuid4
 from unittest.mock import patch
 
 from fastapi.testclient import TestClient
 from unittest.mock import patch
```

Ruff (0.14.0)

Redefinition of unused `patch` from line 2

Remove definition: `patch`

(F811)